dkarlovi/symfony-flex-docker/0.1/Makefile

```diff
@@ -0,0 +1,51 @@
+# see https://github.com/dkarlovi/symfony-flex-docker
+
+APP_RUNNER_ID=82
+
+docker-start:
+	docker-compose -f ./docker-compose.yaml -f ./.infra/docker-compose/docker-compose.$(APP_ENV).yaml up -d
+.PHONY: docker-up
+
+docker-stop:
+	docker-compose down
+.PHONY: docker-down
+
+docker-start-dev:
+	docker-compose -f ./docker-compose.yaml -f ./.infra/docker-compose/docker-compose.dev.yaml up -d
+.PHONY: docker-start-dev
+
+docker-start-prod:
+	docker-compose -f ./docker-compose.yaml -f ./.infra/docker-compose/docker-compose.prod.yaml up -d
+.PHONY: docker-start-prod
+
+docker-start-test:
+	docker-compose -f ./docker-compose.yaml -f ./.infra/docker-compose/docker-compose.test.yaml up -d
+.PHONY: docker-start-test
+
+docker-build:
+	docker-compose build
```

Comment:
There is no "environment-specific build", the idea is to use **exactly the same images on production** you use in development, by pushing them (from your CI server) to a [Docker registry](https://docs.docker.com/registry/) and pulling them from production (say, Kubernetes or Openshift).

The environment-specific Compose files are for runtime-only, not for build.